### PR TITLE
COMPLETE, Kev G

### PR DESCRIPTION
**Pull request by TheObserverofOne**

### CODE REVIEW

**Comment by jryanp1** [comment]
OK, good progress. There is some horizontal scrollbar action at 892 px width, and also services product etc headers show up in the top left corner, so if you want it to look different then look into that. There are no console errors.

**Comment by TheObserverofOne** [comment]
Ah, forgot that media query!  I'll resubmit with the fix.

File: less/index.less [hunk 1]
```diff
@@ -62,114 +68,244 @@ h1, h2, h3, h4, h5 {
...
+                    text-decoration: underline;
+                    color: black;
+
+                    @media @smartphones {
+                        background: gray;
+                    }
+                }
+            }
+
+            &:first-child {
```
Comment (lefrenk) [inline less/index.less]: Excellent! 👌

File: less/index.less [hunk 2]
```diff
@@ -62,114 +68,244 @@ h1, h2, h3, h4, h5 {
...
+            }
+        }
+    }
+
+    .logo {
+        align-self: flex-end;
+        margin: 28px 0 0 61px;
+
+        @media @tablets {
+            order: -1;
```
Comment (lefrenk) [inline less/index.less]: Very nice!

File: less/index.less [hunk 3]
```diff
@@ -62,114 +68,244 @@ h1, h2, h3, h4, h5 {
...
-    align-items: center;
-    margin: 0 82px 0 0;
+    @media @tablets {
+        margin-top: 50px;
+        flex-flow: column;
+        align-items: center;
+    }
+
+    .cta-text {
+        font-size: 72px;
```
Comment (lefrenk) [inline less/index.less]: Try and be consistent with your choice of units for `font-size`. You used `rem` in other sections

**Comment by lefrenk** [comment]
Hey Kev! Everything looks fantastic at each breakpoint. 👌

- You've demonstrated understanding of nested selectors and media queries, less operators, and variables
- Code formatting looks great!

Nothing much to add. Keep up the good work 🙌 Please close this PR when you've reviewed the comments. Thanks!

**Pull request closed without merging**